llama_stack/providers/remote/inference/vllm/vllm.py

```diff
@@ -270,6 +270,12 @@ async def chat_completion(
         tool_config: Optional[ToolConfig] = None,
     ) -> AsyncGenerator:
         model = await self.model_store.get_model(model_id)
+        # This is to be consistent with OpenAI API and support vLLM <= v0.6.3
+        # References:
+        #   * https://platform.openai.com/docs/api-reference/chat/create#chat-create-tool_choice
+        #   * https://github.com/vllm-project/vllm/pull/10000
+        if (tools is None or len(tools) == 0) and tool_config is not None:
```

Comment:
nit: `if not tools and tool_config is not None` or `if tool_config and not tools` is shorter

Reply:
Thanks!